plugins/adrv9002.c

```diff
@@ -185,7 +184,7 @@ static adrv9002_config lte_defaults(void)
 
 	tx_radio_channel_config tx_config[2];
 	int i;
-	for(i = 0; i < CHANNEL_COUNT; i++) {
+	for(i = 0; i < 2; i++) {
```

Comment:
ADRV9002_NUM_CHANNELS...

Reply:
ADRV9002_NUM_CHANNELS is not yet defined here. The fix would be to move the profile gen structures below the other defines. Is it worth it?

Reply:
the fix should be moving the functions below the defines. I'm even guessing that function is only called during init? If so, move it close that function and reduce it's scope...